### ISSUE TITLE
nvda does not read virus warning slideup alerts of avira antivirus

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4768]
**Reported by enes on 2015-01-02 15:38**
hi,
since version 10 of avira antivirus, the virus warning was changed from a dialogue box into a slideup desktop alert.  This alert is totally inaccessible with nvda. However, when a sighted person hovers over the alert with a mouse, nvda reads the controls of the alert. Note that I have reported this to avira numerous times.  I also told them how the issue could be fixed on their end.  
see
https://answers.avira.com/en/question/accessibility-issues-with-avira-28692

**Comment by leonardder** [comment]
@bhavyashah: As we most likely can't contact the author of this issue, would you be able to give a status update?

**Comment by bhavyashah** [comment]
@leonardder Unfortunately, I do not use Avira anymore. Hence, I will be unable to provide a status update on this issue. However, if there are a bunch of Avira specific issues on the tracker (most of which would probably have been filed by me), I could probably seek assistance from an existing Avira user in testing these by asking on the NVDA Users e-group.

**Comment by Adriani90** [comment]
@enes

**Comment by Adriani90** [comment]
We do not have any updates from Avira users since more than three years. I am closing this for now, given the fact that the interface changed significantly in the last years. If you still have problems with the last version, please comment and we can reopen the discussion. Thanks.
